components/Start/StartButton/index.js

```diff
@@ -0,0 +1,34 @@
+import styled from "styled-components";
+import Link from "next/link";
+
+export default function StartButton() {
+  return (
+    <StartLink href="/quiz" alt="StartLink">
```

Comment:
The `alt` attribute is only used for images, as far as I know.

```suggestion
    <StartLink href="/quiz">
```